prime-router/src/main/kotlin/fhirengine/engine/LookupTableValueSet.kt

```diff
@@ -40,8 +40,9 @@ class LookupTableValueSet
             throw SchemaException("Value column not found in specified lookup table")
         }
 
+        val filterTable = lookupTable.table.selectColumns(configData.keyColumn, configData.valueColumn)
```

Comment:
```suggestion
        val filterTable = lookupTable.table.selectColumns(configData.keyColumn, configData.valueColumn)
```

This might avoid an explicit copy (already addressed)